### ISSUE TITLE
Can I have more than one Prebid Network Partner?

### ISSUE THREAD

**Issue by SV-Champ** [prebid/Prebid.js#9819]
## Type of issue

Question

## Description

I have many partners, they want add more connect with their Prebid.js

Can I have more than one Prebid.js or not?

**Comment by patmmccann** [comment]
Yes; just change the global to run simultaneous instances of the library
